src/server.h

```diff
@@ -1402,7 +1402,6 @@ struct sharedObjectsStruct {
 
 /* ZSETs use a specialized version of Skiplists */
 typedef struct zskiplistNode {
-    sds ele;
```

Comment:
Add a comment somewhere in this struct definition, for example after the `level[]` array where the embedded element is located in memory, explaining that there is an embedded sds header length (1 byte) and an embedded sds string here.